src/lv_message_efficient.h

```diff
@@ -93,7 +96,7 @@ namespace grpc_labview
             if (numElements != 0)
             {
                 auto messageTypeSize = sizeof(MessageType);
-                NumericArrayResize(GetTypeCodeForSize(messageTypeSize), 1, reinterpret_cast<void*>(const_cast<char*>(_lv_ptr)), numElements);
+                NumericArrayResize(GetTypeCodeForSize(messageTypeSize), 1, reinterpret_cast<void*>(_lv_ptr), numElements);
```

Comment:
FYI: I don't think you need to explicitly cast to `void*`, since the NumericArrayResize wrapper accepts `void*` and `int8_t*` is convertible to `void*`.